browser/tor/tor_profile_manager.cc

```diff
@@ -88,6 +88,11 @@ Profile* TorProfileManager::GetTorProfile(Profile* original_profile) {
   tor_profile->AddObserver(this);
   tor_profiles_[context_id] = tor_profile;
 
+  tor::TorProfileService* service =
+      TorProfileServiceFactory::GetForContext(tor_profile);
+  DCHECK(service);
+  service->RegisterTorClientUpdater();
```

Comment:
We also need to remove the one in `TorProfileManager::SwitchToTorProfile`

Reply:
Fixed in 7f3183a46cd64fe9b4584338fc17b2d29f026bfe.